### ISSUE TITLE
Are py-pyqt6 deps correct?

### ISSUE THREAD

**Issue by Neumann-A** [open-vcpkg/python-registry#19]
From PySide it know that all qt deps were needed since everything was basically optional. 
Is PyQt6 really only limited to the qt modules mentioned in the manifest? cc @m-kuhn

**Comment by m-kuhn** [comment]
I think you are right, it works the same way @Neumann-A

**Comment by Neumann-A** [comment]
That means you need to add all qt modules as a dependency or add features for every module.

**Comment by m-kuhn** [comment]
> That means you need to add all qt modules as a dependency or add features for every module.

Yes, I planned to work on that whenever I get to using that on a real world project.
